include/mbedtls/ssl_internal.h

```diff
@@ -1454,6 +1454,21 @@ static inline unsigned int mbedtls_ssl_conf_get_endpoint(
 }
 #endif /* MBEDTLS_SSL_CONF_ENDPOINT */
 
+#if !defined(MBEDTLS_SSL_CONF_TRANSPORT)
+static inline unsigned int mbedtls_ssl_conf_get_transport(
+    mbedtls_ssl_config  const *conf )
```

Comment:
```suggestion
    mbedtls_ssl_config const *conf )
```

Reply:
That's been copy-pasted to 20+ places in the file. Shall I fix them all, or just have this one correct+inconsistent?

Reply:
Good point, please just fix your code. I'll create a PR for the rest.